### PR TITLE
[Workers] how the cache works revision

### PR DESCRIPTION
**Pull request by deadlypants1973**

### CODE REVIEW

File: products/workers/src/content/learning/how-the-cache-works.md [hunk 1]
```diff
@@ -5,61 +5,63 @@ pcx-content-type: concept
...
+By allowing developers to write to the cache, Workers provide a way to customize cache behavior on Cloudflare’s CDN. To learn about the benefits of caching, refer to the Learning Center’s article on [What is Caching?](https://www.cloudflare.com/learning/cdn/what-is-caching/).
 
-Since Cloudflare’s Workers can run before, and after the cache, a Worker can also be utilized to modify assets once they are returned from the cache, to sign or personalize responses, while reducing load on an origin, or latency to the end user by serving assets from a nearby location.
+Since Cloudflare Workers can run before, and after the cache, a Worker can also be utilized to modify assets once they are returned from the cache, to sign or personalize responses, while reducing load on an origin, or latency to the end user by serving assets from a nearby location.
 
 --------------------------------
 
 ## Interacting with the Cloudflare Cache
 
 Conceptually, there are two ways to interact with Cloudflare’s Cache using a Worker:
```
Comment (deadlypants1973) [inline products/workers/src/content/learning/how-the-cache-works.md:18]: @lukeed is only Cloudflare Cache and Cache API capitalized but when referring to the concept of a cache it is lowercase?

File: products/workers/src/content/learning/how-the-cache-works.md [hunk 2]
```diff
@@ -5,61 +5,63 @@ pcx-content-type: concept
...
-- __New namespaces__ – You can access a namespaced cache (separate from the cache shared with `fetch` requests) using `let cache = await caches.open(CACHE_NAME)`. This can be useful when writing new content to the cache, for example after running a more compute heavy operation such as parsing HTML or running a computation, to store them locally in the colo and readily access them on the following request, rather having to rerun the same operation. Note that [`caches.open`](https://developer.mozilla.org/en-US/docs/Web/API/CacheStorage/open) is an async function, unlike `caches.default`.
+- __New namespaces__ – You can access a namespaced cache (separate from the cache shared with `fetch` requests) using `let cache = await caches.open(CACHE_NAME)`. This can be useful when writing new content to the cache, for example, after running a more compute heavy operation such as parsing HTML or running a computation, to store them locally in the colocation and readily access them on the following request, rather having to rerun the same operation. Note that [`caches.open`](https://developer.mozilla.org/en-US/docs/Web/API/CacheStorage/open) is an async function, unlike `caches.default`.
```
Comment (deadlypants1973) [inline products/workers/src/content/learning/how-the-cache-works.md]: @lukeed 2 things.

1. I feel like "new namespace" should be replaced with "cache.open" to be parallel with the previous "caches.default header. How do you think?

2. Everything after the first sentence is run-on sentence vibes... Does it make sense for you?

File: products/workers/src/content/learning/how-the-cache-works.md [hunk 3]
```diff
@@ -5,61 +5,63 @@ pcx-content-type: concept
...
 
 
 When to use the Cache API:
 
-- When you need to read from cache without calling fetch. (i.e. send me the response from `slow.com/resource` if and only if it’s already a HIT on cache using `caches.default.match(..)`.
-- Explicitly store a response in the cache using `caches.default.put(..)` and explicitly delete `caches.default.delete(..)`. For example, say your origin is returning `max-age:0` and you can’t figure out how to change that header at your origin. You can explicitly tell Cloudflare to cache this response by setting `cache-control: max-age=1000` on the response passed into `cache.put()`.
+- When you need to read from cache without calling `fetch`. (For example, send me the response from `slow.com/resource` if and only if it is already a HIT on cache using `caches.default.match(..)`).
```
Comment (deadlypants1973) [inline products/workers/src/content/learning/how-the-cache-works.md]: @lukeed

File: products/workers/src/content/learning/how-the-cache-works.md (hunk 2 above)
Comment (deadlypants1973) [inline products/workers/src/content/learning/how-the-cache-works.md]:
```suggestion
- __`caches.open()`__ – You can access a namespaced cache (separate from the cache shared with `fetch` requests) using `let cache = await caches.open(CACHE_NAME)`. Note that [`caches.open`](https://developer.mozilla.org/en-US/docs/Web/API/CacheStorage/open) is an async function, unlike `caches.default`.
```

File: products/workers/src/content/learning/how-the-cache-works.md [hunk 4]
```diff
@@ -5,61 +5,63 @@ pcx-content-type: concept
...
+- Explicitly store a response in the cache using `caches.default.put(..)` and explicitly delete `caches.default.delete(..)`. For example, say your origin is returning `max-age:0` and you cannot figure out how to change that header at your origin. You can explicitly tell Cloudflare to cache this response by setting `cache-control: max-age=1000` on the response passed into `cache.put()`.
```
Comment (lukeed) [inline products/workers/src/content/learning/how-the-cache-works.md]:
```suggestion
- When you want to programmatically save and/or delete responses from a cache. For example, say an origin is responding with a `Cache-Control: max-age:0` header and cannot be changed. Instead, you can clone the `Response`, adjust the header to the `max-age=3600` value, and then use the Cache API to save the modified `Response` for an hour.

- When you want to programmatically access a Response from a cache without relying on a `fetch` request. For example, you can check to see if you have already cached a `Response` for the `https://example.com/slow-response` endpoint. If so, you can avoid the slow request.
```

File: products/workers/src/content/learning/how-the-cache-works.md [hunk 5]
```diff
@@ -5,61 +5,64 @@ pcx-content-type: concept
 
 # How the Cache works
 
-Since Workers was built atop Cloudflare’s network, since its early days, it was designed to allow developers to interact directly with the Cloudflare cache. The cache can provide ephemeral, colo-local storage, as a convenient way to frequently access static or dynamic content.
+Workers was designed and built on top of Cloudflare's edge network to allow developers to interact directly with the Cloudflare cache. The cache can provide ephemeral, colocation-local storage, as a convenient way to frequently access static or dynamic content.
```
Comment (crwaters16) [inline products/workers/src/content/learning/how-the-cache-works.md]: Would normally suggest turning "colocation" into "data center" but understand why you left it in here - will let you make the final call

File: products/workers/src/content/learning/how-the-cache-works.md [hunk 6]
```diff
@@ -5,61 +5,64 @@ pcx-content-type: concept
...
-- __New namespaces__ – You can access a namespaced cache (separate from the cache shared with `fetch` requests) using `let cache = await caches.open(CACHE_NAME)`. This can be useful when writing new content to the cache, for example after running a more compute heavy operation such as parsing HTML or running a computation, to store them locally in the colo and readily access them on the following request, rather having to rerun the same operation. Note that [`caches.open`](https://developer.mozilla.org/en-US/docs/Web/API/CacheStorage/open) is an async function, unlike `caches.default`.
+- __New namespaces__ – You can access a namespaced cache (separate from the cache shared with `fetch` requests) using `let cache = await caches.open(CACHE_NAME)`. This can be useful when writing new content to the cache, for example, after running a more compute heavy operation such as parsing HTML or running a computation, to store them locally in the colocation and readily access them on the following request, rather having to rerun the same operation. Note that [`caches.open`](https://developer.mozilla.org/en-US/docs/Web/API/CacheStorage/open) is an async function, unlike `caches.default`.
```
Comment (crwaters16) [inline products/workers/src/content/learning/how-the-cache-works.md]: data center

**Review by crwaters16 — APPROVED** [review]

File: products/workers/src/content/learning/how-the-cache-works.md (hunk 6 above)
Comment (deadlypants1973) [inline products/workers/src/content/learning/how-the-cache-works.md]:
```suggestion
- __`caches.open()`__ – You can access a namespaced cache (separate from the cache shared with `fetch` requests) using `let cache = await caches.open(CACHE_NAME)`. Note that [`caches.open`](https://developer.mozilla.org/en-US/docs/Web/API/CacheStorage/open) is an async function, unlike `caches.default`.
```

File: products/workers/src/content/learning/how-the-cache-works.md (hunk 5 above)
Comment (deadlypants1973) [inline products/workers/src/content/learning/how-the-cache-works.md]:
```suggestion
Workers was designed and built on top of Cloudflare's edge network to allow developers to interact directly with the Cloudflare cache. The cache can provide ephemeral, data center-local storage, as a convenient way to frequently access static or dynamic content.
```

**Pull request merged**